libs/css-codegen/src/lib/style-utils.ts

```diff
@@ -0,0 +1,40 @@
+/**
+ * This cleans the 'number part' of a length value (ex: 5px) to obtain 1 digit rounded values
+ * @param value css value to be converted
+ */
+export function convertValue(value: any): string {
+  return value.toString().split(' ').map(convert).join(' ');
+}
+
+/**
+ * This cleans the 'number part' of a length value (ex: 5px) to obtain 1 digit rounded values
+ * @param value css value to be converted
+ */
+function convert(value: string): string {
+  const numberMatches = value.match(/[-+]?[0-9]*\.?[0-9]+([eE][-+]?[0-9]+)?/g);
+  if (numberMatches) {
+    let convertedValue = value.slice();
+    numberMatches.forEach(
+      (found) =>
+        (convertedValue = convertedValue.replace(found, getCleanNumber(found)))
+    );
+    return convertedValue;
+  }
+  return value;
+}
+
+function getCleanNumber(number: any) {
+  if (!isNaN(number)) {
+    if (number.includes('e')) {
+      number = Math.exp(parseFloat(number));
+    } else {
+      number = parseFloat(number);
+    }
+    if (number % 1 != 0) {
```

Comment:
```suggestion
    if (number % 1 !== 0) {
```

Reply:
I see that one of the errors from our CI is this related to this change. When the change has been pushed it will be gone. 😀

Reply:
@anediaz just as an FYI, we have strict linting rules for xlayers when it comes to double/triple equals (see https://github.com/xlayers/xlayers/blob/main/tslint.json#L53) ^^